src/User/src/Entity/UserResetPasswordEntity.php

```diff
@@ -11,11 +11,7 @@
 use Doctrine\ORM\Mapping as ORM;
 use Throwable;
 
-/**
- * @ORM\Entity()
- * @ORM\Table(name="user_reset_password")
- * @ORM\HasLifecycleCallbacks()
- */
+#[ORM\Entity, ORM\Table(name: "user_reset_password"), ORM\HasLifecycleCallbacks]
```

Comment:
```suggestion
#[ORM\Entity]
#[ORM\Table(name: "user_reset_password")]
#[ORM\HasLifecycleCallbacks]
```
Just for better readability.